Saint-Nec-Engine/CommonLib/Component.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+class Component // Abstraite ? On manipule forc�ment une sous classe
+{
+
+public:
+	Component() = default;
+	~Component() = default;
+	virtual void update() = 0;
+	virtual void draw() const = 0;
+};
+
+// Si pas de r�el besoin, ajouter les classes filles ici ?
```

Comment:
@antonindevidal Si les classes filles ne sont pas 'énorme' et que tu es OK, on laisse tout dans le même fichier ?

Reply:
They will be big, I just didn't think of what will be in there, so they need to be on different files